search/controllers/api/__init__.py

```diff
@@ -216,15 +234,17 @@ def _get_include_fields(params: MultiDict, query_terms: List) -> List[str]:
     return []
 
 
-def _get_fielded_terms(params: MultiDict, query_terms: List) \
-        -> Optional[FieldedSearchList]:
+def _get_fielded_terms(params: MultiDict, query_terms: List,
```

Comment:
Why is this being changed? The classic queries can't be translated to the "fielded search term" model, so I'm confused about why this code is involved.

Reply:
Still curious about this

Reply:
This is only being used in the present representation of NG search in `search.controllers.api.search`. Classic queries use `search.controllers.api.classic_query`